outputs.tf

```diff
@@ -8,12 +8,27 @@ output "runner_cache_bucket_arn" {
   value       = module.cache.arn
 }
 
-output "runner_agent_role" {
+output "runner_cache_bucket_name" {
+  description = "Name of the S3 for the build cache."
+  value       = module.cache.bucket
+}
+
+output "runner_agent_role_arn" {
   description = "ARN of the rule used for the ec2 instance for the GitLab runner agent."
   value       = aws_iam_role.instance.arn
 }
 
-output "runner_role" {
+output "runner_agent_role_name" {
+  description = "Name of the rule used for the ec2 instance for the GitLab runner agent."
+  value       = aws_iam_role.instance.name
+}
+
+output "runner_role_arn" {
   description = "ARN of the rule used for the docker machine runners."
   value       = aws_iam_role.docker_machine.arn
 }
+
+output "runner_role_name" {
+  description = "Name of the rule used for the docker machine runners."
+  value       = aws_iam_role.docker_machine.arn
```

Comment:
The ARN is returned instead of the name. Name of variable not matching the return value